spec.md

```diff
@@ -245,6 +243,32 @@ querying of the Service Broker's catalog, the Platform might not have an
 end-user with which to associate the request, therefore in those cases the
 originating identity header would not be included in those messages.
 
+## Service Broker Errors
+
+When a request to a Service Broker fails, the Service Broker MUST return an
+appropriate HTTP response code. If an appropriate response code is not defined
+within this specification, Service Broker authors SHOULD chose a code that
+accurately reflects the nature of the failure.
+
+For error responses, the following fields are valid:
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| error | string | A single word uniquely identifying the error condition. |
+| description | string | A meaningful error message explaining why the request failed. | 
```

Comment:
I think in other spots we talk about thing being "user-facing" to imply it should be human readable, perhaps we could use that wording since "meaningful" could be interpreted as "meaningful to a machine so its easily parse-able".

Reply:
Done!